src/elements/paginator/paginator.ts

```diff
@@ -129,6 +132,15 @@ export class SbbPaginatorElement extends SbbNegativeMixin(LitElement) {
     }
   }
 
+  protected override firstUpdated(_changedProperties: PropertyValues): void {
```

Comment:
To align with our normal code style, you could remove "_" from changedProperties